### PR TITLE
feat(evaluation): support multi datasets

### PR DESCRIPTION
**Pull request by goldenxinxing**
## Description

1. sever: Support multi datasets, the model's relation are as below:
session -> consumer -> dataset
     1        :         n         :       m
2. client: In order to ensure the uniqueness of data under multiple datasets, add uri as prefix for data id/index
image

4. client: add multi dataset parm for the cmd  "swcli eval run" and "swcli model eval" 

## Modules
- [ ] UI
- [x] Controller
- [ ] Agent
- [x] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [x] add unit test
- [ ] add necessary doc

### CODE REVIEW

**Review by jialeicui — APPROVED** [review]
LGTM

File: client/starwhale/core/model/cli.py [hunk 1]
```diff
@@ -159,14 +159,15 @@ def _extract(model: str, force: bool, target_dir: str) -> None:
...
+    multiple=True,
     help=f"dataset uri, env is {SWEnv.dataset_uri}",
 )
 def _eval(
     project: str,
     target: str,
     model_yaml: str,
     version: str,
-    dataset: str,
+    dataset: list,
```
Comment (tianweidut) [inline client/starwhale/core/model/cli.py:170]: dataset -> datasets or dataset_uris?

File: server/controller/src/main/java/ai/starwhale/mlops/StarwhaleControllerApplication.java [hunk 2]
```diff
@@ -23,11 +23,13 @@
 import org.springframework.boot.builder.SpringApplicationBuilder;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
```
Comment (tianweidut) [inline server/controller/src/main/java/ai/starwhale/mlops/StarwhaleControllerApplication.java:26]: remove this import ?

**Comment by tianweidut** [comment]
Others LGTM

File: client/starwhale/core/model/cli.py (hunk 1 above)
Comment (goldenxinxing) [reply client/starwhale/core/model/cli.py]: done

File: server/controller/src/main/java/ai/starwhale/mlops/StarwhaleControllerApplication.java (hunk 2 above)
Comment (goldenxinxing) [reply server/controller/src/main/java/ai/starwhale/mlops/StarwhaleControllerApplication.java]: done

**Review by tianweidut — APPROVED** [review]

**Pull request merged**